python/pyspark/sql/tests.py

```diff
@@ -1851,6 +1851,71 @@ def test_hivecontext(self):
         self.assertIn("default", out.decode('utf-8'))
         self.assertTrue(os.path.exists(metastore_path))
 
+    def test_hivecontext_create_dataframe(self):
```

Comment:
did you mean have the same scenario where multiple SQLContext objects are constructed but the test success criteria is to verify that no more than a single `SparkSession` object ever gets created?